docs/installation.rst

```diff
@@ -35,7 +38,8 @@ You can either clone the public repository:
 
     $ git clone git://github.com/roocs/clisops
 
-Create Conda environment named `clisops`:
+Create Conda environment named `clisops`. For windows users, you might need to edit the `environment` file
+and remove the `xESMF` package.
```

Comment:
What about making a `.. warning::` for Windows users here?

Reply:
It is the way. BRB